### ISSUE TITLE
Can only use once, otherwise the next decoration overrides the first widget decoration. very bizarre

### ISSUE THREAD

**Issue by josephvisaac** [kalaganov/corner_decoration#2]

**Comment by kalaganov** [comment]
do you have an example so I can reproduce it?

**Comment by josephvisaac** [comment]
I discovered why. I fixed it but this was the issue: your package only
works properly inside cards, it can be duplicated easily in cards, but if
done directly in a container you can only have one use with it, and the
previous code disappears. I'm using it very well now inside cards, I can
set it easily to make the card look like any container. I love your package.

**Comment by singleseeker** [comment]
```
Row(
                        crossAxisAlignment: CrossAxisAlignment.start,
                        mainAxisAlignment: MainAxisAlignment.spaceBetween,

**Comment by singleseeker** [comment]
![2020-11-13 13 54 15](https://user-images.githubusercontent.com/520698/99034313-d7851c80-25b7-11eb-9bb1-3a41f2524085.gif)
````
Row(
                        crossAxisAlignment: CrossAxisAlignment.start,
                        mainAxisAlignment: MainAxisAlignment.spaceBetween,    
                        children: [
                              Container,
                              Container,
                         ]
````
When using the plugin in the first container, the second container will disappear

**Comment by singleseeker** [comment]
@kalaganov any update？

**Comment by singleseeker** [comment]
@josephvisaac busy guy, any update?

**Comment by kalaganov** [comment]
hello guys!
sorry for late answer
I think I found an issue and published new version 1.0.2

try it please, if something will still wrong please leave a message here

**Comment by singleseeker** [comment]
wow, great job. 

thank you very much.

![image](https://user-images.githubusercontent.com/520698/99191271-b6365300-27a6-11eb-99ff-672433f25ec1.png)
